content/logs/aws.md

```diff
@@ -0,0 +1,122 @@
+Using a Lambda function, called whenever a new S3 log file is made available, avoids the hassle of creating processes that would pull periodically incoming S3 logs.
+
+## Setup
+### Installation
+#### Create a new lambda function
+
+1. Navigate to the [Lambda Console](https://console.aws.amazon.com/lambda/home?region=us-east-1) and create a new function:
+    {{< img src="logs/aws/create_lambda_function.png" alt="Create Lambda function" >}}
+
+2. Select **Author from scratch** and give the function a unique name.
+    {{< img src="logs/aws/author_from_scratch.png" alt="Author from Scratch" >}}
+3. For `Role`, select **Create new role from template(s)** and give the role a unique name
+4. Under Policy templates, search for and select **s3 object read-only permissions**
+    {{< img src="logs/aws/basic_information.png" alt="Basic Information" >}}
+5. Select **Create Function**
+
+#### Provide the code and configure the lambda
+
+6. Copy and paste the code from [This repo](https://github.com/DataDog/dd-aws-lambda-functions/blob/master/Log/lambda_function.py) into the function code area.
+7. Change the Runtime to **Python 2.7**
+    {{< img src="logs/aws/select_python.png" alt="Select Python" >}}
+8. At the top of the script you'll find a section called `#Parameters`, that's where you want to edit your code.
+    {{< img src="logs/aws/dd_api_key_setup.png" alt="DD API key setup" >}}
+Two options for this, either:
+
+    * Directly in the code 
+    * Use an environment variable called `DD_API_KEY`.
+
+9. Set the memory to the highest possible value.
+10. Set the timeout limit. We recommend **120 seconds**
+    {{< img src="logs/aws/basic_settings.png" alt="Basic Settings" >}}
+11. Scroll back to the top of the page and hit **Save and test**
+
+#### Test your Lambda
+
+12. Configure the test event. Select **Cloudwatch Logs** as the sample event.
+    {{< img src="logs/aws/test_event.png" alt="Test Event" >}}
+13. Save and test.
+
+## Collection
+
+Your lambda function is now ready to send logs to your Datadog platform. All you need is to setup the relevant triggers for each AWS service you want to monitor.
+
+#### S3, Cloudwatch, API Gateway, Kinesis and SNS
+In your lambda, go in the triggers tab and select `Add Trigger`:
+{{< img src="logs/aws/adding_trigger.png" alt="Adding trigger" >}}
+
+Select the log source and then follow the AWS instructions: 
+{{< img src="logs/aws/integration_lambda.png" alt="Integration Lambda" >}}
+
+For instance for S3 Buckets, do not forget to set the correct event type:
+{{< img src="logs/aws/object_created.png" alt="Object Created" >}}
+
+#### ELB
+
+Elastic Load Balancing provides access logs that capture detailed information about requests or connections sent to your load balancer. Each log contains information such as the time it was received, the client's IP address, latencies, request paths, and server responses. Use these access logs to analyze traffic patterns and troubleshoot issues.
```

Comment:
each service has an introduction paragraph. I think these belong in the intro at the top. Keep this section just about the setup instructions.

Reply:
agreed 👍